lib/tasks/emails.rake

```diff
@@ -1,12 +1,12 @@
 namespace :emails do
   desc "Count the number of users who have received a survey"
-  task :count_2021_03_survey_recipients, %i[] => [:environment] do |_, _args|
+  task :count_2021_03_survey_recipients, %i[] => [:environment] do |_, args|
```

Comment:
`args` isn't used in this task.

Reply:
oh true, sorry got carreid away, just the one below then.